tests/test_multiline_variables.py

```diff
@@ -66,6 +70,4 @@ def test_multiline_variable_stdin(tmp_path):
     del env['ARGUMENT']
 
     result = subprocess.run(["python3.7", script_path, "-a", "-"], **common_args, input="", check=False)
-    assert result.returncode != 0
-    assert "IncorrectParameterError" in result.stderr
-    assert error_message in result.stderr
+    assert error_message in result.stdout
```

Comment:
Why have we removed error type check?

Reply:
Previously run of a test script result in an exception.
The script is changed to print error explicitly and it doesn't raise the exception.